samples/net/mqtt/doc/description.rst

```diff
@@ -120,20 +120,8 @@ In addition, the sample provides the following overlay configuration files, whic
 
 They are located in :file:`samples/net/mqtt` folder.
 
-To add a specific overlay configuration file to the build, add the ``-- -DOVERLAY_CONFIG=<overlay_config_file>`` flag to your build.
-
-See :ref:`cmake_options` for instructions on how to add this option to your build.
-For example, when building with the command line, the following commands can be used for the nRF9160 DK:
-
-.. code-block:: console
-
-   west build -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-tls-nrf91.conf
-
-For Thingy:91, with TLS and debug logging enabled for the :ref:`lib_mqtt_helper` library (for more information, see the related :ref:`sample output <mqtt_sample_output_IPv6>`):
-
-.. code-block:: console
-
-   west build -b thingy91_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-tls-nrf91.conf -DCONFIG_MQTT_HELPER_LOG_LEVEL_DBG=y
+To add a specific overlay configuration file to the build, set the :makevar:`EXTRA_CONF_FILE` option to the given file name.
```

Comment:
Here also I'd like to have at least one example command before linking to more info.